SDBLV/beliveo-files
# Remove duplicate CodeSniffer rules

#### What does this PR do?
- Delete CodeSniffer rules duplicate

#### Where should the reviewer start?
- beliveo-files/config/rules/SDStandard/

#### How should this be manually tested? (if applies)
:one: in a app, get this repository
:two: modify php file (with syntax errors)
:three: in the bash or terminal -> npm run sniff
:four: should show errors
:five: fix errors
:six: run point 3
:seven: should no longer show errors
